com.carpeludum.KegaFusion.yml

```diff
@@ -0,0 +1,119 @@
+app-id: com.carpeludum.KegaFusion
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
+sdk: org.freedesktop.Sdk
+command: Fusion
+tags:
+  - proprietary
+finish-args:
+   # X11 + XShm access
+   - --share=ipc
+   - --socket=x11
+   # Play sounds
+   - --socket=pulseaudio
+   # Joystick and GPU access
+   - --device=all
+   # Get access to the files
+   - --filesystem=home:ro
+   - --filesystem=xdg-run/gvfs:ro
+   # Run 32-bit software
+   - --allow=multiarch
+
+build-options:
+  prepend-pkg-config-path: /app/lib32/pkgconfig:/usr/lib/i386-linux-gnu/pkgconfig
+  ldflags: -L/app/lib32
+  append-path: /usr/lib/sdk/toolchain-i386/bin
+  env:
+    CC: i686-unknown-linux-gnu-gcc
+    CXX: i686-unknown-linux-gnu-g++
+  libdir: /app/lib32
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Compat.i386
+  - org.freedesktop.Sdk.Extension.toolchain-i386
+
+add-extensions:
+  org.freedesktop.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: "20.08"
+
+  org.freedesktop.Platform.Compat.i386.Debug:
+    directory: lib/debug/lib/i386-linux-gnu
+    version: "20.08"
+    no-autodownload: true
+
+  org.freedesktop.Platform.GL32:
+    directory: lib/i386-linux-gnu/GL
+    version: '1.4'
+    versions: "20.08;1.4"
+    subdirectories: true
+    no-autodownload: true
+    autodelete: false
+    add-ld-path: lib
+    merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d;OpenCL/vendors;lib/dri;lib/d3d;vulkan/explicit_layer.d
+    download-if: active-gl-driver
+    enable-if: active-gl-driver
+
+
+modules:
+
+  - shared-modules/glu/glu-9.json
+  - shared-modules/gtk2/gtk2.json
+
+  - name: kega-fusion
+    buildsystem: simple
+    build-commands:
+      - make install
+      - install -D fusion.sh /app/bin/Fusion
+      - install -D Fusion /app/lib/KegaFusion/Fusion
+      - install -D Fusion.ini /app/lib/KegaFusion/Fusion.ini
+      - install -d /app/lib/KegaFusion/Plugins/
+      - cp *.rpi /app/lib/KegaFusion/Plugins/
+      - install -d /app/share/doc/KegaFusion
+      - cp *.txt /app/share/doc/KegaFusion
+      - install -Dm644 com.carpeludum.KegaFusion.png /app/share/icons/com.carpeludum.KegaFusion.png
+      - install -Dm644 -t /app/share/applications/ com.carpeludum.KegaFusion.desktop
+      - install -Dm644 -t /app/share/metainfo/ com.carpeludum.KegaFusion.metainfo.xml
+      # Directories for the extensions
+      - mkdir -p /app/lib/i386-linux-gnu•
+      - mkdir -p /app/lib/debug/lib/i386-linux-gnu•
+      - mkdir -p /app/lib/i386-linux-gnu/GL•
+      - install -Dm644 -t /app/etc ld.so.conf
+
+    sources:
+      - type: archive
```

Comment:
I think the question here remains whether it's redistributable, but if it's not being actively developed, I guess it can stay that way.

Reply:
I don't have an answer. I'm fairly certain it is redistributable, and has been redistributed, and I'm trying to get in touch with the original author.

Reply:
The developer seems to be fine the software being distributed without restrictions. The software hasn't been updated since 2009, so I doubt we'll ever see an update with a license file for example.